test/tigrc/segfault-test

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+
+. libtest.sh
+
+test_todo "Segfault without 'set reference-format' in tigrc"
```

Comment:
Is this a test for the todo marker or is it something you plan to fix in a future PR?